test/formatting.coffee

```diff
@@ -198,6 +198,27 @@ test "#1495, method call chaining", ->
   ).join ', '
   eq 'a, b, c', result
 
+test "chaining after inline ternary", ->
+  throws -> CoffeeScript.compile """
+    if 0 then 1 else g
+      a: 42
+    .h()
+  """
+
+  # Wrap calls containing spilling ternary
```

Comment:
Please put a new line like `test "wrap calls containing spilling ternary", ->` to start a new test.

Currently if anything inside this `test` function fails, the whole test fails and it’s hard to know what part of this failed, since you’re testing multiple things here (the `throws` and the two `eq`s). It would be nice if each of those were separate `test` functions, so that if one of them fails we know exactly which one.